python/datashaper/tests/test_schema.py

```diff
@@ -25,42 +25,28 @@
 
 
 @cache
-def load_inputs(pandas_dtype_backend: PandasDtypeBackend, engine: str):
+def load_inputs():
     dataframes: dict[str, pd.DataFrame] = {}
+
     for file in os.listdir(TABLE_STORE_PATH):
+        path = Path(TABLE_STORE_PATH) / file
         if file.endswith(".csv"):
-            path = Path(TABLE_STORE_PATH) / file
-            table = pd.read_csv(
-                path,
-                dtype_backend=pandas_dtype_backend,
-                engine=engine,
-            )
+            table = read_csv(path)
             dataframes[file.removesuffix(".csv")] = table
+        if file.endswith(".json") and file != "workflow.json":

```

Comment:
We're doing a lot of `endswith` in this PR - maybe it's worth a filename extension utility in datashaper/core or datashaper/tables?

Reply:
Yeah, good point - we _do_ have that in the js impl, py is just way behind on file loading stuff